docs/ref/settings.txt

```diff
@@ -963,6 +963,23 @@ TEST_TBLSPACE_TMP
     Use the :setting:`TBLSPACE_TMP <TEST_TBLSPACE_TMP>` entry in the
     :setting:`TEST <DATABASE-TEST>` dictionary.
 
+.. setting:: DATA_UPLOAD_MAX_MEMORY_SIZE
+
+DATA_UPLOAD_MAX_MEMORY_SIZE
+---------------------------
+
+.. versionadded:: 1.8
```

Comment:
Also, please update the settings index: https://docs.djangoproject.com/en/dev/ref/settings/#core-settings-topical-index